modules/swffg-main.js

```diff
@@ -614,6 +614,23 @@ function isCurrentVersionNullOrBlank(currentVersion) {
 Hooks.once("ready", async () => {
   SettingsHelpers.readyLevelSetting();
 
+  if (!game.settings.get("starwarsffg", "token_configured")) {
+    const tokenData = {
+      bar1: {
+        attribute: "stats.wounds",
+      },
+      bar2: {
+        attribute: "stats.strain",
+      },
+      displayBars: 30, // hovered by anyone
```

Comment:
I know speaking personally I wouldn't want hovered by anyone as the default for showing strain and wounds... in my games I tend to leave it as hovered by owner to avoid metagaming.

Reply:
I personally leave it here, as it doesn't display numbers - just overall percent of health lost - and that's something (IMO) you can get by looking at someone. That being said, it's just the default. It can be changed.

THAT being said, I also don't care what the value is. If you want "hovered by owner", I'm fine with that and can update it.

Reply:
I'd definitely prefer hovered by owner as the default

Reply:
Okay, updated.